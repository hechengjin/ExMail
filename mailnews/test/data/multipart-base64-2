Content-Type: multipart/mixed; boundary=boundary

yadda yadda

--boundary
Content-Type: text/html
Content-Transfer-Encoding: base64

PGh0bWw+PGhlYWQ+YmFzZTY0ZW5jb2RlZCBIVE1MIHRleHQgaW5zaWRlIGEgbXVsdGlwYXJ0IG1lc3N
hZ2UuPC9oZWFkPjwvaHRtbD4=

--boundary--
