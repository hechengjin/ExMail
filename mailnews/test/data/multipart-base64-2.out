<html><head>base64encoded HTML text inside a multipart message.</head></html>